### PR TITLE
adding support for lodestarfinance to the roundtable list

### PR DESCRIPTION
**Pull request by Cooops**

### CODE REVIEW

**Comment by Cooops** [comment]
heyo -- any update here by chance? happy to tweak anything if need be :)

**Comment by Cooops** [comment]
@0xMyrddin bump good sir, let me know if you need anything else from us <3

**Comment by 0xMyrddin** [comment]
Apologies missed that one, will add a few comments

File: src/protocols/arbitrum-one.json [hunk 1]
```diff
@@ -345,5 +345,30 @@
...
+    "cover": "https://github.com/Lodestar-Finance/lodestar-assets/blob/main/Lodestar_Banner_1500x500px_v1.png",
+    "tagline": "Lodestar: Arbitrum's Native Money Market",
+    "tags": [
+      "yield",
+      "lending",
+      "DAO"
+    ],
+    "description": "Lodestar Finance is an algorithmic borrowing and lending protocol that brings the critical DeFi primitive of decentralized money markets to Arbitrum communities.",
+    "tokens": [
+      "0xf19547f9ed24aa66b03c3a552d181ae334fbb8db"
```
Comment (0xMyrddin) [inline src/protocols/arbitrum-one.json:363]: Need checksummed addresses only

File: src/protocols/arbitrum-one.json [hunk 2]
```diff
@@ -345,5 +345,30 @@
     "github": "https://github.com/Sperax",
     "telegram": "https://t.me/SperaxUSD",
     "defillama": "https://defillama.com/protocol/sperax"
+  },
+  {
+    "chainId": 42161,
+    "id": "lodestar",
+    "name": "Lodestar",
+    "logo": "https://raw.githubusercontent.com/Lodestar-Finance/lodestar-assets/main/Frame_237.png",
+    "cover": "https://github.com/Lodestar-Finance/lodestar-assets/blob/main/Lodestar_Banner_1500x500px_v1.png",
```
Comment (0xMyrddin) [inline src/protocols/arbitrum-one.json:354]: This url won't work (should use raw.*)

File: src/protocols/arbitrum-one.json (hunk 1 above)
Comment (LodestarFinance) [reply src/protocols/arbitrum-one.json]: @0xMyrddin updated! im sorry, now I forget about this PR 👯

File: src/protocols/arbitrum-one.json (hunk 2 above)
Comment (LodestarFinance) [reply src/protocols/arbitrum-one.json]: @0xMyrddin updated!

**Pull request merged**